app/views/layouts/_application_content.html.erb

```diff
@@ -15,6 +15,15 @@
         <div class="btn-toolbar pull-right"><%=h yield(:title_actions) %></div>
       </div>
     </div>
+    <div  class="row">
+      <div class='col-md-6' id="breadcrumb">
+        <% if content_for?(:breadcrumbs) %>
+          <%= yield(:breadcrumbs) %>
+        <% else %>
+          <%= mount_breadcrumbs(get_resource) if !searchable? && get_resource %>
```

Comment:
Should we really use searchable as a way to define breadcrums ? even if its the same condition I would prefer to have a dedicated method.

I'm also not sure that if there is a search component we should not show breadcrums.. ?

Reply:
+1 for having a dedicated method.
I have no strong opinion about checking the `!searchable?` we can tune it later if it doesn't work 100%.